Topic recognition

lucaslin2020 opened the pull request.

lucaslin2020 committed:
Create README.md

lucaslin2020 committed:
Add files via upload

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Add files via upload

lucaslin2020 committed:
Add files via upload

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Delete validandtest.ipynb

lucaslin2020 committed:
Add files via upload

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Add files via upload

lucaslin2020 committed:
Delete _recall.txt

lucaslin2020 committed:
Delete _result.txt

lucaslin2020 committed:
Delete map.txt

lucaslin2020 committed:
Add files via upload

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Rename map4000.txt to mapother.txt

lucaslin2020 committed:
Rename map.txt to map4000.txt

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

nathasha-naranpanawa commented:
- Please remember to provide a detailed description for the pull-request
- Poor commit messages
- Code is okay for normal difficulty
- Performance is somewhat poor - has not achieved 0.8 IoU

Other notes
- Using LabelImg for ground truth bounding box annotations is unacceptable

lucaslin2020 committed:
Update README.md

lucaslin2020 committed:
Update README.md

New version

lucaslin2020 commented:
Dear Teacher,I have provude some description and commit messages. Also, delete the content about Labelling in readme. Thanks for your feedback.Best Regards,Runqi Lin-------- 原始邮件 --------发件人： Nathasha Naranpanawa ***@***.***>日期： 2021年11月16日周二 15:22收件人： shakes76/PatternFlow ***@***.***>抄送： lucaslin2020 ***@***.***>, Author ***@***.***>主    题： Re: [shakes76/PatternFlow] Topic recognition (PR #261)
Please remember to provide a detailed description for the pull-requestPoor commit messagesCode is okay for normal difficultyPerformance is somewhat poor - has not achieved 0.8 IoU
Other notes
Using LabelImg for ground truth bounding box annotations is unacceptable

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

lucaslin2020 committed:
Update README.md

Change the hyperparameters in yolov3.cfg and train a new model to get the better performance.

lucaslin2020 committed:
Add files via upload

Update new result file about new hyperparameters

shakes76 commented:
## TF/Torch Usage
Adequate use and implementation
## Good Practice (Design/Commenting)
Good spacing and comments
Header blocks
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present but as notebooks, you a module for functions/models -1
No Data leakage
Difficulty: Normal -5
## Commit Log
Meaningful commit messages, could be more descriptive -1
Progressive commits used
## Documentation
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
No feedback required
Request Description missing -2
